### PR TITLE
Add arrow keys to move controls on GUI Editor

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

File: guiEditor/src/diagram/workbench.tsx [hunk 1]
```diff
@@ -975,6 +975,25 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
                 case "F":
                     this.globalState.onFitToWindowObservable.notifyObservers();
                     break;
+                case "ArrowUp": // move up
+                case "W":
+                case "w":
+                    this.moveControls(false, k.event.shiftKey ? -5 : -1);
```
Comment (darraghjburke) [inline guiEditor/src/diagram/workbench.tsx:981]: Can we make this a constant?

File: guiEditor/src/diagram/workbench.tsx [hunk 2]
```diff
@@ -1015,6 +1034,34 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         this.props.globalState.onGizmoUpdateRequireObservable.notifyObservers();
     }
 
+    // Move the selected controls. Can be either on horizontal (leftInPixels) or 
+    // vertical (topInPixels) direction
+    moveControls(horizontal: boolean, amount: number) {
```
Comment (darraghjburke) [inline guiEditor/src/diagram/workbench.tsx:1039]: should second parameter be named vertical?

**Review by darraghjburke — COMMENTED** [review]
love it!! just a couple nitpicks

File: guiEditor/src/diagram/workbench.tsx (hunk 2 above)
Comment (RaananW) [reply guiEditor/src/diagram/workbench.tsx:1039]: seems like horizontal is the mode (i.e. horizontal when true, vertical when false?).

**Review by RaananW — APPROVED** [review]
LGTM! I agree with @darraghjburke - the numbers can be constants (to avoid repeating them), but apart from that - all good :-)

Approved, waiting for resolution of the comments already provided.

File: guiEditor/src/diagram/workbench.tsx (hunk 2 above)
Comment (carolhmj) [reply guiEditor/src/diagram/workbench.tsx:1039]: Yeah, you're right, it's the mode! I think `moveHorizontal` could work, what do you think?

Comment (carolhmj) [reply guiEditor/src/diagram/workbench.tsx]: Updated!

File: guiEditor/src/diagram/workbench.tsx (hunk 1 above)
Comment (carolhmj) [reply guiEditor/src/diagram/workbench.tsx]: Did it!

File: guiEditor/src/diagram/workbench.tsx (hunk 2 above)
Comment (darraghjburke) [reply guiEditor/src/diagram/workbench.tsx]: Ah got it, I misread it, thanks for the explanation!

**Pull request merged**